Detectors/ITSMFT/ITS/workflow/src/DCSParserSpec.cxx

```diff
@@ -149,6 +161,20 @@ void ITSDCSParser::updateMemoryFromInputString(const std::string& inString)
     return;
   }
   std::string doubleColsEORstr = inString.substr(pos);
+
+  // In this case the terminator is missing and the DCOLS_EOR field is empty:
+  // the stave is passed whitout meaningful infos
+  if (doubleColsEORstr == "|") {
+    this->writeChipInfo(this->mConfigDCS, this->mStaveName, -1);
```

Comment:
If you want, for `writeChipInfo` you can also avoid passing mConfigDCS since it's an object visible to all member functions.